### PR TITLE
Fix currency language dropdown style

### PR DESCRIPTION
**Pull request by mahmed0715**
@sagararyal please check this. we tried to implement react select for both..

### CODE REVIEW

File: app/components/Currency/GlobalCurrencySelector.js [hunk 1]
```diff
@@ -51,20 +100,46 @@ class GlobalCurrencySelector extends Component {
...
+            value: this.state.preferredCurrency,
+            label: this.state.preferredCurrency
+          }}
+          menuPlacement="top"
+          styles={customStyles}
+          onChange={this.handleCurrencyChange}
+          isSearchable={false}
+          options={currenciesArray}
+        />
+        {/* <div>
```
Comment (norbertschuler) [inline app/components/Currency/GlobalCurrencySelector.js]: We don’t need to keep old commented code. We have Git 😁

Comment (mahmed0715) [reply app/components/Currency/GlobalCurrencySelector.js]: mistake

**Comment by norbertschuler** [comment]
Tested on latest Chrome on MacOS - I am not so convinced by the design, e.g. the language menu looks like it's cut at the right side where the background color suddenly ends with the text instead of filling the whole row:

Bildschirmfoto 2019-10-15 um 12 03 59

Also I think the font size is to large and should be the same as the font in the menu items:

Bildschirmfoto 2019-10-15 um 12 04 07

**Comment by mahmed0715** [comment]
Okay. will change.

**Comment by mahmed0715** [comment]
Fixed those issues. font size is same as other menu items(dont be confused about the capital case for currency, it looks bigger but i checked its same font size)

**Comment by mahmed0715** [comment]
@norbertschuler please review and merge this

**Pull request merged**